exercises/bank.js

```diff
@@ -1,51 +1,83 @@
-
-
-class Bank{
-	constructor(){
+class Bank {
+	constructor(accountName) {
+		this.name = accountName;
+		this.storage = {};
 		//takes in the name of the bank.
 		//makes an object to store all accounts into
 	}
-	makeAccount( ){
+	makeAccount(accountNumber) {
```

Comment:
Functions that are included in a "class constructor" are called methods, and will be attached to any object that is created by the "class constructor". This is incredibly useful for easily adding functionality to objects with the ease of an assembly line. Imagine creating a "person class constructor" that needed to have methods such as "eating", "drinking", and "studying". By doing this, every student that was created by the "class constructor" would be able to eat, drink, and study (I would say "sleep" would be a good method, but most students don't get to do that). 
The utility of this is easy to understand, as it allows code to be easily replicated in each new object simply by calling the "class constructor" Great Job!